### ISSUE TITLE
Add js library to load and display NIFTI images

### ISSUE THREAD

**Issue by Angeluz-07** [Angeluz-07/integradora_lesiones_cerebrales#9]
Tasks : 
1. Investigate about a javascript library to load a NIFTI image and display it.
2. Investigate how to implement this tool in the project. See if possible to integrate it with out current basic django project, or if it is needed to set up a UI-Client  project(with a node and a UI framework). Ideally we would want to integrate it as a single js library.
3. If possible to get a single js library, add it to the project with very basic example view.

**Comment by Angeluz-07** [comment]
After some research. I found the following two options, that could be useful : 
- https://github.com/rii-mango/Papaya ([Demo](https://papaya.greenant.net/))
- https://github.com/aces/brainbrowser ([Demo](https://brainbrowser.cbrain.mcgill.ca/volume-viewer))

Refs:
https://f.giorlando.org/2018/07/web-viewers-for-fmri/

By inspection, I see `Papaya` could be easier to integrate. I will start with checking out that, and see what happens.

**Comment by Angeluz-07** [comment]
To integrate library, I navigated through project and found assets ready to use in the folder `https://github.com/rii-mango/Papaya/tree/build-1449/release/current/standard`

With these assets, I have tested that is possible to achieve the following, with this library + django:
- Load Papaya.js as a standalone js library in the django project.
- Load `.nii.gz` images through the browser and visualize them immediately.
- Preload a `.nii.gz` image from server.

**Comment by Angeluz-07** [comment]
Regarding to load a `.nii.gz` image from the server, I must say that some special procedure needs to be followed to make it work with Django(ar at least the way we make it work):
1. Serve the `.nii.gz` image  via `static`. i.e. if we are running the project locally the file must be available at `http://localhost:8000/static/example_mri.nii.gz`. See django docs for details.
2. With javascript, fetch the file via HTTP request, decode it as String Base64 and pass it to Papaya instance.

Basic illustration example:
```js
var URL_MRI = "{% static '/example_mri.nii.gz' %}"; // django way to build url of static files
var params = [];

//https://stackoverflow.com/questions/6150289/how-can-i-convert-an-image-into-base64-string-using-javascript
const toDataURL = url => fetch(url)
    .then(response => response.blob())
    .then(blob => new Promise((resolve, reject) => {
        const reader = new FileReader()
        reader.onloadend = () => resolve(reader.result)
        reader.onerror = reject
        reader.readAsDataURL(blob)
    }))

//Fetch and decode
toDataURL(URL_MRI)
.then(dataUrl => {
    console.log('RESULT:', dataUrl)
    var imgEncodedAsStringBase64 = dataUrl.split(';base64,')[1];//Get only the string encoded content
    params["encodedImages"] = [imgEncodedAsStringBase64];
    papaya.Container.resetViewer(0, params);// set the new image, see Papaya.js docs for more details.
})

```

**Comment by Angeluz-07** [comment]
Closing Remarks:
- We will proceed with Papaya.js as our viewer tool. 
- We saw, is possible integrate it with Django as a standalone js library(with some manageable tweaks).
- As implementation of this issue, I will add only assets. In another issue I will add the final implementation.

**Comment by Angeluz-07** [comment]
Fixed in #10
